src/ts/utils/roam.ts

```diff
@@ -58,7 +63,80 @@ export const Roam = {
         await Keyboard.pressEnter(15);
         await Keyboard.pressEnter();
         document.execCommand('paste')
+    },
+
+    moveCursorToStart() {
+        this.applyToCurrent(
+            node =>
+                new RoamNode(
+                    node.text,
+                    // new Selection(node.text.length * 2, node.text.length * 2)
+                    new Selection(0,0)
+                ))
+    },
+
+    moveCursorToEnd() {
+        this.applyToCurrent(
+            node =>
+                new RoamNode(
+                    node.text,
+                    new Selection(node.text.length * 2, node.text.length * 2)
+                ))
+    },
+
+    writeText(text: string) {
+        this.applyToCurrent(node => 
+            new RoamNode(text, node.selection));
+    },
+
+    async createSiblingAbove() {
+        this.moveCursorToStart();
+        const isEmpty = !this.getActiveRoamNode()?.text;
+        await Keyboard.pressEnter(20);
+        if (isEmpty) {
+            await Keyboard.simulateKey(38,20); //Up Arrow
+        }
+    },
+    
+    async createSiblingBelow() {
+        this.moveCursorToEnd();
+        await Keyboard.pressEnter(20);
+        await Keyboard.pressShiftTab(40);
```

Comment:
why 40? Also we probably want to extract `20` to a constant

Reply:
as with the others, trial and error... 20 was fine when testing the command itself, but when building the demo, I was getting weird results and this solved it.

@panpiotrs mentioned using `MutationObserver`, and that it could make things slow, but I assume that meant having it _always-on_ watching for user interactions... if we only use it when executing the script, shouldn't be a problem. Something to test in the next iterations.

Reply:
I see. Making constant alongside one with 20?

Reply:
I believe you created `standardDelay` for that already, so I changed it to 20.

Reply:
standardDelay was intentionally 0, but I'm not sure if it's a right decision or if it should be 20. 

With your latest change there is no way to specify anything less then 20 though, which I find worrying. As I said above - I'm not sure what standard one should be, but if it's not 0 then the passed in value should probably be not additional delay but delay override

Reply:
makes sense - see if it's better now.

by the way just tested the demo with 0 delay and worked... for a few times, and on a blank page only, then broke again. There's no magic number since it's heavily dependant on page size - which is obvious given we really feel the delay when editing a long page.

don't want to spend too much time tinkering this delay thing since it's clear the other option must be better. 

For now, 20 seems to work even for longer pages. And now can be overriden if needed